pkgs/tools/misc/babashka/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, lib, graalvm8, fetchurl }:
```

Comment:
Normally we put fetchurl right after stdenv:

`{ stdenv, fetchurl, lib, graalvm8 }:`